src/controllers/expense.controller.js

```diff
@@ -0,0 +1,102 @@
+const expenseService = require('../services/expense.service');
+const userService = require('./../services/user.service');
+
+const getExpenses = async (req, res) => {
+  const { userId, categories, from, to } = req.query;
+
+  try {
+    const expenses = await expenseService.getAllExpense({
+      userId,
+      categories,
+      from,
+      to,
+    });
+
+    res.status(200).send(expenses);
```

Comment:
Consider adding more detailed error messages or logging the error details in the catch block to help with debugging and monitoring.